include/task.h

```diff
@@ -1,6 +1,10 @@
 #ifndef GUARD_TASK_H
 #define GUARD_TASK_H
 
+// general task defines
+#define TASK gTasks[taskId]
+#define FUNC func
```

Comment:
It says multiple declarations of the same declaration but continues compiling. It's not really a big deal but I want this in a global file goddamnit.